### PR TITLE
Bump version to 5.7

### PR DESCRIPTION
**Pull request by WilliamBZA**

### CODE REVIEW

File: src/Custom.Build.props [hunk 1]
```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <MinVerMinimumMajorMinor>5.4</MinVerMinimumMajorMinor>
+    <MinVerMinimumMajorMinor>5.6</MinVerMinimumMajorMinor>
```
Comment (ramonsmits) [inline src/Custom.Build.props:18]: Shouldn't master be bumped to 5.7?

Comment (ramonsmits) [reply src/Custom.Build.props:18]: Maybe we should ask David/Brandon for a review?

**Comment by ramonsmits** [comment]
Currently builds on master have the wrong tag:

- https://github.com/Particular/ServiceControl/pkgs/container/servicecontrol

Last is `5.6.0-alpha.0.19` but that likely should have been something like `5.7.0-alpha.0.1`.

File: src/Custom.Build.props (hunk 1 above)
Comment (ramonsmits) [inline src/Custom.Build.props:18]: @DavidBoike @bording Shouldn't master be bumped to 5.7?

```suggestion
    <MinVerMinimumMajorMinor>5.7</MinVerMinimumMajorMinor>
```

**Review by bording — APPROVED** [review]

**Pull request merged**